ts/Core/Series/Series.ts

```diff
@@ -663,6 +663,29 @@ class Series {
         );
     }
 
+    /**
+     * Determine whether the marker in a series has changed.
+     *
+     * @private
+     * @function Highcharts.Series#hasMarkerChanged
+     */
+    public hasMarkerChanged(
+        options: DeepPartial<SeriesOptions>,
+        oldOptions: DeepPartial<SeriesOptions>
+    ): boolean | undefined {
+        const series = this,
+            marker = options.marker,
+            oldMarker = oldOptions.marker || {};
+
+        // This is a workaround for update lowMarker symbol #14024
```

Comment:
It's not really a "workaround", more a refactoring to allow extendibility. I think this comment can just be removed.